src/coins.ts

```diff
@@ -558,7 +558,7 @@ const basicCoins: BasicCoin[] = [
         decimals: 6,
       },
       [ChainId.MOO]: {
-        address: '0x8f552a71efe5eefc207bf75485b356a0b3f01ec9',
+        address: '0x818ec0a7fe18ff94269904fced6ae3dae6d6dc0b',
```

Comment:
It looks like there are multiple USDC tokens on Moonriver which are all used. The one defined `0x8f552a71efe5eefc207bf75485b356a0b3f01ec9` is the one used by the connext bridge. We should not change that.

If you require another token for stellaswap, please define it in the stellaswap configuration only.

Reply:
okay, gotcha.
I will update it soon.